relative-json-pointer.xml

```diff
@@ -125,8 +125,9 @@
                 Evaluation begins by processing the non-negative-integer prefix.
                 This can be found by taking the longest continuous sequence of decimal
                 digits available, starting from the beginning of the string, taking
-                the decimal numerical value.  If this value is more than zero, then
-                the following steps are repeated that number of times:
+                the decimal numerical value. If the value is zero, the following stepps
```

Comment:
typo "stepps"

might be better state the general case, which applies to zero as well, then note the behavior on zero. "The following steps are repeated the number of times indicated as the prefix value. If this value is zero, the steps are not performed."

Reply:
```suggestion
                the decimal numerical value. If the value is zero, the following steps
```